### ISSUE TITLE
Use a function for repeated tasks

### ISSUE THREAD

**Issue by sen1** [fearandwonder/intro-data-capstone-biodiversity#1]
https://github.com/fearandwonder/intro-data-capstone-biodiversity/blob/84dc9edecb1c757d28f0e8179bcde7ee60430a48/Codecademy_Intro-to-Data-Analysis_Capstone-Project_S-Houghton/capstone_project_code.py#L78-L222

You have so many same lines of codes there. Also, another inefficient approach is hard coding the values. If I changed the dataset you will have to redo the numbers again. You have the power of python to make it smarter, put the below in a function and pass only the pivot table:
```
from itertools import combinations
from scipy.stats import chi2_contingency
for c in combinations(range(len(category_pivot)), 2):
    pick2comb = category_pivot.iloc[list(c),:]
    print(pick2comb)
    print(chi2_contingency(pick2comb.iloc[:,1:3]))
```

**Comment by fearandwonder** [comment]
Yes, this was horribly inefficient - as I was writing it I was having difficulty keeping track of which combinations i had left to do! itertools is a library I have no knowledge of and I thank you for the snippet and alerting me to this - hopefully I can use more efficient coding in the future!
